### PR TITLE
C# built-in types: mention dynamic as reference type

### LINKED ISSUE #18849: Why No Dynamic as Reference Types?
Hello, is dynamic considered as reference type? if yes, i saw no dynamic as part of reference type?

thank you, correct me if i was wrong
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fe245daf-f6b8-1100-3786-b15feefe137b
* Version Independent ID: c031bb57-cb5b-f953-4711-a0ed054dab3f
* Content: [Built-in types - C# reference](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/built-in-types)
* Content Source: [docs/csharp/language-reference/builtin-types/built-in-types.md](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/builtin-types/built-in-types.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by pkulikov**
Fixes #18849

### CODE REVIEW

File: docs/csharp/language-reference/builtin-types/built-in-types.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The following table lists the C# built-in [reference](../keywords/reference-type
 |--------------|-------------------------|
 |[`object`](reference-types.md#the-object-type)|<xref:System.Object?displayProperty=nameWithType>|
 |[`string`](reference-types.md#the-string-type)|<xref:System.String?displayProperty=nameWithType>|
+|[`dynamic`](reference-types.md#the-dynamic-type)||
```
Comment (pkulikov) [inline docs/csharp/language-reference/builtin-types/built-in-types.md:36]: Should there be any text in the empty cell of the ".NET type" column?

Comment (BillWagner) [reply docs/csharp/language-reference/builtin-types/built-in-types.md:36]: The dynamic type doesn't map to a different .NET type. I'd put `<xref:System.Object>` as well.

**Review by BillWagner — COMMENTED** [review]
this looks great once we update that table cell.

File: docs/csharp/language-reference/builtin-types/built-in-types.md (hunk 1 above)
Comment (pkulikov) [reply docs/csharp/language-reference/builtin-types/built-in-types.md]: Thank you @BillWagner, I've updated the table.

Comment (pkulikov) [reply docs/csharp/language-reference/builtin-types/built-in-types.md]: @BillWagner ping

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**